packages/client/hmi-client/src/page/project/components/tera-resource-sidebar.vue

```diff
@@ -15,17 +15,7 @@
 				<!-- A way to use vue feather icons in the MenuItem component, it might be better to try to use 1 icon library for easier integration -->
 				<template #item="slotProps">
 					<a class="p-menuitem-link">
```

Comment:
Why not using your new `tera-asset-link`?

Reply:
The `tera-asset-link` just has the functionality to open an existing asset and that's it (not to mention the styles are different as well).  For this in particular, it's within the PrimeVue Menu component which handles it own actions to create _new_ assets.

<img width="396" alt="Screenshot 2023-10-20 at 1 26 32 PM" src="https://github.com/DARPA-ASKEM/terarium/assets/33158416/dc0b638a-7da3-4cfc-b3b9-94dacc2dd3c4">

Reply:
I see what you mean...